docs/how-to/convert-react-app.md

```diff
@@ -0,0 +1,24 @@
+---
+parent: How To
+nav_order: 1
+title: Convert React App to Modular Project
+---
+
+# Convert a React App to a Modular Project
+
+To convert an existing React app project into a Modular project, start by
+creating a new Modular project:
+
+` yarn create modular-react-app my-new-modular-project [--verbose] [--prefer-offline] [--repo]`
+
+Once created:
+
+- Replace the source code within the App workspace inside the `packages/`
```

Comment:
Within the `app` workspace, so it's clearer that we're talking about a workspace called `app`.